src/mixins/filing-template-mixin.ts

```diff
@@ -414,6 +415,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     // restore Business Number
     this.setRegistrationBusinessNumber(draftFiling.registration.business.taxId || null)
 
+    // restore Business Type Confirm
+    this.setRegistrationBusinessTypeConfirm(draftFiling.registration.businessTypeConfirm || false)
```

Comment:
I changed null to false because otherwise it's initial value being null is giving me an error in the unfinished step/scrolling down mechanics.

Reply:
Does the error/scrolling mechanism still work for a Sole Prop? Maybe something special needs to be done in there so it only applies to GPs.
Here, I still prefer null since it's neither false nor true, so we consider it correct for a SP. Another option is to wrap this code in an if statement so it's only set for GPs.

Reply:
Fixed